config/routes.md

```diff
@@ -6,6 +6,9 @@ routes:
     url: /
     options:
       hero: true
+      description:
+        fr: Coopérative qui conçoit et développe des services numériques d’intérêt général.
+        en: Cooperative that designs and develops digital services in the public interest.
```

Comment:
alternative : A cooperative dedicated to designing and developing digital services in the public interest